docs/io/optional/logging_configuration.ipynb

```diff
@@ -0,0 +1,2226 @@
+{
```

Comment:
<p>Change the comma to a colon.&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>